spec/system/supervisors/index_spec.rb

```diff
@@ -70,32 +65,34 @@
       let(:expected_first_ordered_value) { "11" }
       let(:expected_last_ordered_value) { "9" }
 
-      it "by supervisor name", js: true do
-        expect(page).to have_selector("th.sorting_desc", text: "Supervisor Name")
-        expect(page).to have_selector("tr:nth-child(1)", text: "Last Supervisor")
+      it "by supervisor name", :aggregate_failures, js: true do
```

Comment:
Is :aggregate_failures just for debugging here?

Reply:
It is just to group more than one failure by `it` block. 
https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures